proto/autoops/command.proto

```diff
@@ -103,3 +103,31 @@ message AddProgressiveRolloutTemplateScheduleClauseCommand {
 message ChangeProgressiveRolloutScheduleTriggeredAtCommand {
   string schedule_id = 1;
 }
+
+message CreateFlagTriggerCommand {
+  string feature_id = 1;
+  int32 type = 2;
+  int32 action = 3;
```

Comment:
Please can you use the enum value instead of int32 for the type and action fields?